File: A4.java
package ejercicios;

public class Ejercicios {

    public static void main(String[] args) {
        String x = "La vaca pasta en el campo azulino";
        x = x.replaceAll("[aA]+", "@");
        x = x.replaceAll("[eE]+", "3");
        x = x.replaceAll("[iI]+", "1");
        x = x.replaceAll("[oO]+", "X");
        x = x.replaceAll("[uU]+", "V");
        System.out.println(x);
    }

}

File: Reemplazo de vocales en una frase/A4.py
import re
x = "La vaca pasta en el campo azulino"


x = re.sub("[aA]+","@", x)
x = re.sub("[eE]+", "3", x)
x = re.sub("[iI]+", "1", x)
x = re.sub("[oO]+", "X", x)
x = re.sub("[uU]+", "V", x)


print(x)